docs/source/beta.rst

```diff
@@ -98,244 +110,298 @@ If your system meets the minimum required kernel version you can proceed with th
 
-There are several configurable options that must be considered:
+Selecting the Quipucords Server Configuration Options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When you run the command to start the Quipucords server, you supply values for several options that affect the configuration of that server. You must make the following decisions:
 
-- Exposed server port
+- Accepting or changing the default exposed server port
 - Selecting a directory for SSH keys
 - Selecting a directory for the SQLlite database
 - Selecting a directory for log output
 
+The following steps guide you through those choices.
 
-The server exposes port 443, which is the standard HTTPS port. You may choose to utilize that port or re-map the port on your server.
+1. Accept or change the default exposed server port to use for HTTPS communication. By default, the server exposes port 443, which is the standard HTTPS port. You can choose to use that port or remap the port to be used on your server.
 
-If you selected to expose port 443 then you would use the following option when running the image ``-p443:443``. If you wish to re-map the port on your system Docker's mapping is -p<host_port>:<container_port>. If you choose for example to re-map the port to 8443 the option to supply would be ``-p8443:443``. Additionally, Docker supplies an option to select a free port for all exposed ports by using the ``-P`` option; the port mapping is then available from the ``docker ps`` command.
+   - If you select to expose port 443, you would use the following option when you run the Docker command to start the server: ``-p443:443``.
+   - If you want to remap the port on your system, you would supply a new value for port when you run the Docker command to start the server. The syntax of this option is  ``-p<host_port>:<container_port>``. For example, to remap the port to ``8443``, you would enter the followng option in the command: ``-p8443:443``. Additionally, Docker supplies an option to select a free port for all exposed ports by using the ``-P`` option; the port mapping is then available from the ``docker ps`` command.
 
+2. Select values for the directory for SSH keys, the directory for the SQLlite database, and the directory for the log output. The most efficient way to configure these options is to create a home directory for the Quipucords server and then use that home directory for each of thse three options.
 
-For the next three configuration options we will take a simple setup strategy for the Quipucords server and create a "home directory" for the server.
+   \a. Create the home directory. The following example command creates the home directory  ``/opt/quipucords``:
```

Comment:
The latest changes had moved this to the `~/quipucords` directory to avoid the user having to use `sudo` to create directories in the `opt` directory.